gcp/website/frontend_handlers.py

```diff
@@ -534,8 +534,19 @@ def osv_query(search_string, page, affected_only, ecosystem):
 
   bugs, _, _ = query.fetch_page(
       page_size=_PAGE_SIZE, offset=(page - 1) * _PAGE_SIZE)
-  for bug in bugs:
-    result_items.append(bug_to_response(bug, detailed=False))
+
```

Comment:
Add a comment/TODO for this check to say that it doesn't work properly if there are multiple pages.